cmd/ll2go/instruction.go

```diff
@@ -314,6 +318,26 @@ func (d *decompiler) instGetElementPtr(inst *ir.InstGetElementPtr) ast.Stmt {
 	return d.assign(inst.Name, expr)
 }
 
+func (d *decompiler) instExtractValue(inst *ir.InstExtractValue) ast.Stmt {
+	src := d.value(inst.X)
+	for _, index := range inst.Indices {
+		src = &ast.IndexExpr{
+			X:     src,
+			Index: d.int(index),
+		}
+	}
+	return d.assign(inst.Name, src)
+}
+
+func (d *decompiler) instExtractElement(inst *ir.InstExtractElement) ast.Stmt {
+	src := &ast.IndexExpr{
+		X:     d.value(inst.X),
+		Index: d.value(inst.Index),
+	}
+	return d.assign(inst.Name, src)
+}
```

Comment:
Thanks! The implementation of both `extractvalue` and `extractelement` looks good.